frontend/src/component/events/EventDiff/EventDiff.tsx

```diff
@@ -12,9 +12,13 @@ const DIFF_PREFIXES: Record<string, string> = {
 
 interface IEventDiffProps {
     entry: Partial<IEvent>;
+    sort?: (a: any, b: any) => number;
```

Comment:
Could we avoid the use of `any` here? I've got the impression that we're generally trying to avoid it in the codebase. How about one of these?

I don't know how the interface is used other places, though, so it might be incovenient 🤷🏼 

Either type param (with a an optional default to avoid causing issues at other places where it's used):

```suggestion
interface IEventDiffProps<T = { key: string }> {
    entry: Partial<IEvent>;
    sort?: (a: T, b: T) => number;
```

Or lock it down to only use the specific type

```suggestion
interface IEventDiffProps {
    entry: Partial<IEvent>;
    sort?: (a: {key: string}, b: {key: string}) => number;
```

wdyt?

Reply:
That's a valid concern, thanks. Addressed in https://github.com/Unleash/unleash/pull/3307/commits/b9e702fe146512908f390b687bae4e99188f47bb.